### ISSUE TITLE
Docker db

### ISSUE THREAD

**Issue by Sblop** [paperwork/paperwork#903]
A quick suggestion. I am trying to use paperwork using docker on my qnap nas. 
First of all thanks for making a docker image available, however it refuses to start due to a lack of an MYSQL db. I have tried with built-in mysql db on nas, or an additional mysql official docker image. Somebody made a docker image including a db several years ago and this works fantastic, however this is outdated.

Long story short: A humble suggestion to make a docker version of paperwork including the DB.

Thanks in advance and have nice weekend ahead.

**Comment by mrusme** [comment]
This issue was reported for v1, which is not in active development anymore. However, for v2 I did just that. Check out the [Docker stack file](https://github.com/paperwork/paperwork/blob/master/paperwork.yml). It allows you to [easily set up your whole v2 infrastructure](https://github.com/paperwork/paperwork#docker-stack).
